### ISSUE TITLE
Missing install DBT cli step or Compose run

### ISSUE THREAD

**Issue by cassiobolba** [Snowflake-Labs/sfquickstarts#766]
in the Airflow + DBT + Snowflake quick start (https://quickstarts.snowflake.com/guide/data_engineering_with_apache_airflow/index.html?index=..%2F..index#1)

there is a step asking to run `dbt init dbt`  but in no previous step is asking to install DBT before, or to run the the docker compose file to install Airflow + DBT.

Also, in the same page it is saying the structure tree should have a docker file. From where?

Please, make the steps more explicits.

**Comment by vamshigudipati** [comment]
I agree. The above issue still remains open. @cassiobolba - Are you able to overcome this issue?

**Comment by sfc-gh-mharding** [comment]
I also agree.

**Comment by cassiobolba** [comment]
I did overcome because I already know how to work with docker and dbt core a little bit. People new to it would not be able to proceed. Sorry, but it has been almost a year so I do not remember the workaround I did

**Comment by rahul1889** [comment]
Hey, 
i faced the same issue, refer this doc till 3rd point 
https://docs.astronomer.io/learn/airflow-dbt

and then go to next step in snowflake project again, which will be https://quickstarts.snowflake.com/guide/data_engineering_with_apache_airflow/index.html?index=..%2F..index#2

**Comment by andresbi** [comment]
Yep, the documentation on this is pretty faulty. I don't like installing things on my local computer, but not sure how to do it with a virtual environment in addition to the docker container
